### PR TITLE
Rafajpet/helm config improvements

### PR DESCRIPTION
**Pull request by rafajpet**

### CODE REVIEW

File: charts/plgd-hub/README.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ global:
 | certificateauthority.config.fileName | string | `"service.yaml"` | File name for config file |
 | certificateauthority.config.mountPath | string | `"/config"` | Mount path |
 | certificateauthority.config.volume | string | `"config"` | Config file volume name |
+| certificateauthority.customPodPorts | object | `{}` | Custom certificate-authority container ports definition. In case it's not defined, default definition is used |
```
Comment (ondrejtomcik) [inline charts/plgd-hub/README.md]: Why won't you use `certificateauthority.service.ports` for this? Is there a reason to have a pod port defined even though you don't have it exposed through the service?

Comment (rafajpet) [reply charts/plgd-hub/README.md]: I try to figure out scenario where you would like to forward traffic from service itself not directly to main container within pod but to different container, let's say auth sidecar. You don't want to break api for ingress/other service consume this service so you don't want to change port for service but within POD you would like to switch port for sidecar with original container port.

Comment (ondrejtomcik) [reply charts/plgd-hub/README.md]: Do you have such a scenario now or is it just a theoretical expectation? Change only what is needed.

Comment (rafajpet) [reply charts/plgd-hub/README.md]: Add custom authorization logic for grpc-gateway, certificate-authority, http-gateway in form of sidecar for those services

Comment (ondrejtomcik) [reply charts/plgd-hub/README.md]: But we don't have custom authorization logic. Do we?

Comment (rafajpet) [reply charts/plgd-hub/README.md]: We would like to use it for kistler IoT platform. Special permission logic for subset of plgd-hub services

**Comment by ondrejtomcik** [comment]
Is this the correct annotation for the HTTP gateway? 
https://github.com/plgd-dev/hub/blob/rafajpet/helm-config-improvements/charts/plgd-hub/values.yaml#L1086

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**